crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -1033,6 +1033,7 @@ impl<'a, 'o> TimelineEventHandler<'a, 'o> {
                 send_state: EventSendState::NotSentYet,
                 transaction_id: txn_id.to_owned(),
                 send_handle: send_handle.clone(),
+                created_at: send_handle.clone().and_then(|h| h.created_at),
```

Comment:
Probably the cloning is not required here:

```suggestion
                created_at: send_handle.created_at,
```

(or `created_at.clone()`, if borrowck complains)

Reply:
So send_handle is an Option so I can't access created_at directly.  And if I try to map and clone, it seems to dislike the Option being mapped (since self.ctx_flow is a reference).  I think the clone is needed?

Reply:
I think the field is `Copy` so it should work with `send_handle.as_ref().and_then(|h| h.created_at)`.